src/pages/MyProfile.jsx

```diff
@@ -1,15 +1,240 @@
-import React from "react";
+import React, {useEffect, useState} from "react";
+import PopupJS from "reactjs-popup";
 import PageLayout from "../components/PageLayout";
 import Header from "../components/Header";
 import Footer from "../components/Footer";
+import api from "../http/api";
+
+import 'reactjs-popup/dist/index.css';
+import hide from "../assets/hide-pass.svg";
+import show from "../assets/show-pass.svg";
+import {useNavigate, useParams} from "react-router-dom";
+import {toast} from "react-toastify";
 
 function MyProfile() {

```

Comment:
Is there need for special component if it is not used anywhere else, you can simply put it between header and footer and it will lower number of components.